client/src/components/Login.vue

```diff
@@ -1,34 +1,59 @@
 <template>
-  <section>
-    <v-dialog v-model="dialog" :fullscreen="mobile" max-width="500px">
-      <template v-slot:activator="{ props }">
-        <v-btn v-bind="props">Login</v-btn>
-      </template>
-      <v-card>
-        <form @submit.prevent="submitForm">
-          <v-card-title>
-            <span class="text-h5">Login</span>
-          </v-card-title>
-          <v-card-text>
+  <v-button>
+  <v-model fullscreen :scrim="false">
+  <v-col>
+  <v-row justify="center">
+    <v-container>
+      <v-row justify="end">
+        <RouterLink to="/"><v-icon>mdi-close</v-icon></RouterLink>
+      </v-row>
+    </v-container>
+  </v-row>
+  
+  <v-row justify="center">
+    <v-img :src="image" :max-width="250"/>
+  </v-row>
+
+  <v-row justify="center">
+        <v-form @submit.prevent="submitForm">
+        <v-container>
+        <v-row justify="center">
+            <span class="text-h3 text-center">Welcome Back</span>
+        </v-row>
+        <v-row justify="center">
+            <span class="text-h6">Login to your existing account</span>
+        </v-row>
+        </v-container>
+        <v-container>
             <v-form>
               <v-text-field v-model="form.email" label="Email"></v-text-field>
               <v-text-field v-model="form.password" label="Password" type="password"></v-text-field>
+              <v-row justify="center">
+                <RouterLink to="/">Forgot Password?</RouterLink>
+              </v-row>
             </v-form>
-          </v-card-text>
-          <v-card-actions>
-            <v-spacer></v-spacer>
-            <v-btn color="primary" @click="submitForm">Submit</v-btn>
-            <v-btn color="primary" @click="closeDialog">Cancel</v-btn>
-          </v-card-actions>
-        </form>
-      </v-card>
-    </v-dialog>
-  </section>
+        </v-container>
+        <v-container class="mt-10">
+            <v-col>
+                <v-row justify="center">
+                <v-btn color="red" @click="submitForm">Login</v-btn>
+                </v-row>
+                <v-row justify="center">
+                  <p>or <RouterLink to="/">Register</RouterLink></p>
+                </v-row>
```

Comment:
needs more spacing between these. The red is the incorrect colour, when you merge main into your branch use class='bg-primaryRed'